data.sql

```diff
@@ -37,5 +37,33 @@ UPDATE animals SET owner_id = 4 WHERE name IN ('Charmander', 'Squirtle', 'Blosso
 UPDATE animals SET owner_id = 5 WHERE name IN ('Angemon', 'Boarmon');
 COMMIT;
 
+INSERT INTO vets (name, age, date_of_graduation) VALUES ('William Tatcher', 45, '2000-4-23');
+INSERT INTO vets (name, age, date_of_graduation) VALUES ('Maisy Smith', 26, '2019-1-17');
+INSERT INTO vets (name, age, date_of_graduation) VALUES ('Stephanie Mendez', 64, '1981-4-4');
```

Comment:
- [ ] As mentioned by the previous reviewer, kindly make sure you fix the `date_of_graduation` for Vet **'Stephanie Mendez'**. The correct date is `1981-05-04` and not `1981-04-04`